src/frontend/src/pages/CreateChangeRequestPage/CreateChangeRequestView.tsx

```diff
@@ -105,6 +105,21 @@ const CreateChangeRequestsView: React.FC<CreateChangeRequestViewProps> = ({
   const projectOptions: { label: string; id: string }[] = [];
   const wbsDropdownOptions: { label: string; id: string }[] = [];
 
+  const customComparator = (option1: { id: string }, option2: { id: string }) => {
+    // Extract the WBS numbers from the options and convert them to a sortable format
+    const wbsNumber1 = parseFloat(option1.id.replace(/-/g, '.'));
```

Comment:
i dont think this is explicitly seperating projects and work packages but it seems to be working but idk 🤷‍♂️